source/renderer/app/containers/wallet/dialogs/paper-wallet-certificate/InstructionsDialogContainer.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import React, { Component } from 'react';
+import { remote } from 'electron';
+import { observer, inject } from 'mobx-react';
+import InstructionsDialog from '../../../../components/wallet/paper-wallet-certificate/InstructionsDialog';
+import type { ActionsMap } from '../../../../actions/index';
+import type { StoresMap } from '../../../../stores/index';
+
+type Props = {
+  stores: any | StoresMap,
+  actions: any | ActionsMap,
+  onClose: Function,
+};
+
+@inject('stores', 'actions') @observer
+export default class InstructionsDialogContainer extends Component<Props> {
+  static defaultProps = { actions: null, stores: null };
+
+  onPrint = () => {
+    const filePath = remote.dialog.showSaveDialog({
+      defaultPath: '~/paper-wallet-certificate.pdf',
```

Comment:
@tomothespian we need to test this `defaultPath` on Windows installer!

Reply:
I agree